client/src/actions/index.js

```diff
@@ -17,7 +17,9 @@ export const submitBlog = (values, file, history) => async (dispatch) => {
     headers: {
       'Content-Type': file.type
     }
-  });
+  }).catch(error =>
```

Comment:
Have a look at how error handling is done with `async/await`
[This](https://javascript.info/async-await) can get you started:)